example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -283,7 +283,7 @@
 				ENABLE_BITCODE = NO;
 				HEADER_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = example/Info.plist;
-				INFOPLIST_KEY_CFBundleDisplayName = "Dev RN PhylloSampleApp";
+				INFOPLIST_KEY_CFBundleDisplayName = "Dev RN SampleApp";
```

Comment:
Any reason to change this?

Reply:
@akshaybhalotia no reason , give me one specific example name for this . itsn't able to show full name when we install example app in iphone so make it short .

Reply:
Phyllo RN Sample App